pkg/kubectl/resource_printer.go

```diff
@@ -270,10 +268,11 @@ func formatEndpoints(endpoints []api.Endpoint) string {
 		return "<none>"
 	}
 	list := []string{}
-	for i := range endpoints {
-		ep := &endpoints[i]
-		list = append(list, net.JoinHostPort(ep.IP, strconv.Itoa(ep.Port)))
-	}
+	//FIXME: What do we want to print, now that endpoints are more complex?
```

Comment:
@ghodss @bgrant0607 - What do we want kubectl to print in a multi-port-services and endpoints world?